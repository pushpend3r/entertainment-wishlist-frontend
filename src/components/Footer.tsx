import { Container } from "react-bootstrap";

function Footer() {
  return (
    <Container fluid>
      <p className="d-flex p-0 m-0 mt-5 pb-3 justify-content-center">Created by Pushpender Singh</p>
    </Container>
  );
}

export default Footer;
