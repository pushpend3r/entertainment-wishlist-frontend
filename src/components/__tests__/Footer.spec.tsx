import { render } from "@testing-library/react";
import Footer from "../Footer";

describe("<Footer /> component", () => {
  test("render", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
