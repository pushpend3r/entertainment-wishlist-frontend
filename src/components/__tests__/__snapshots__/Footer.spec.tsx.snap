// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Footer /> component > render 1`] = `
<div>
  <div
    class="container-fluid"
  >
    <p
      class="d-flex p-0 m-0 mt-5 pb-3 justify-content-center"
    >
      Created by Pushpender Singh
    </p>
  </div>
</div>
`;
